larq/optimizers.py

```diff
@@ -205,6 +206,10 @@ class Bop(tf.keras.optimizers.Optimizer):
     Bop is a latent-free optimizer for Binarized Neural Networks (BNNs) and
     Binary Weight Networks (BWN).
 
+    To use Bop to train a model set `kernel_quantizer=1` for the layers where
+    binarization should be handled by Bop or use custom predicate function in the
+    `CaseOptimizer`.
```

Comment:
Suggestion:

To use Bop to train a model, set `kernel_quantizer=1` for the layers where training and binarization should be handled by Bop. This is the easiest, recommended approach. Alternatively, you can define a custom predicate function for the [`CaseOptimizer`](https://larq.dev/api/optimizers/#caseoptimizer) that captures the set of layers you want to assign to Bop (see Bop's [`is_binary_var()`](https://github.com/larq/larq/blob/master/larq/optimizers.py#L288) for example.)